### PR TITLE
MSA: fix setting of group states

### PR DESCRIPTION
**Pull request by rhaschke**
MSA restricted names of group states to be unique globally, although they only need to be unique within planning groups. This rendered the definition of hand poses "open" for both hands of a bimanual robot impossible. This is fixed.
Additionally, only _active joints_ are considered for the definition of named group states.

### CODE REVIEW

File: moveit_setup_assistant/src/widgets/robot_poses_widget.cpp [hunk 1]
```diff
@@ -409,31 +405,36 @@ void RobotPosesWidget::playPoses()
...
+  edit(ranges[0].bottomRow());
 }
 
 // ******************************************************************************************
 // Edit pose
 // ******************************************************************************************
-void RobotPosesWidget::edit(const std::string& name)
+void RobotPosesWidget::edit(int row)
 {
-  // Remember what we are editing
```
Comment (davetcoleman) [inline moveit_setup_assistant/src/widgets/robot_poses_widget.cpp]: why reduce my commenting?

File: moveit_setup_assistant/src/widgets/robot_poses_widget.cpp [hunk 2]
```diff
@@ -524,37 +514,37 @@ void RobotPosesWidget::loadJointSliders(const QString& selected)
...
-        (*joint_it)->getVariableDefaultPositions(&init_value);
-      }
-      else  // there is already a value in the map
-      {
-        init_value = joint_state_map_[(*joint_it)->getName()];
-      }
+    // Decide what this joint's initial value is
+    if (joint_state_map_.find(joint_model->getName()) == joint_state_map_.end())
+    {
+      // the joint state map does not yet have an entry for this joint
```
Comment (davetcoleman) [inline moveit_setup_assistant/src/widgets/robot_poses_widget.cpp]: This file uses capitalized comments (such as 3 lines above), please stay consistent

Comment (rhaschke) [reply moveit_setup_assistant/src/widgets/robot_poses_widget.cpp]: These lower case comments were introduce by yourself ;-)
I will change them anyway.

Comment (davetcoleman) [reply moveit_setup_assistant/src/widgets/robot_poses_widget.cpp]: ah sorry :)

**Pull request merged**